RELEASE_NOTES.md

```diff
@@ -1,32 +1,26 @@
-<!--
-  A release notes template that should be adapted for every release
-    - release: <v*.*.*>
-    - release branch: <v*.x>
-    - the last release: <v-last> 
-    - the last release branch: <v-last.x>
--->
-
-# Gaia <v*.*.*>  Release Notes 
+# Gaia v19.0.0  Release Notes 
 
 ## 📝 Changelog
 
-Check out the [changelog](https://github.com/cosmos/gaia/blob/<v*.*.*>/CHANGELOG.md) for a list of relevant changes or [compare all changes](https://github.com/cosmos/gaia/compare/<v-last>...<v*.*.*>) from last release.
+Check out the [changelog](https://github.com/cosmos/gaia/blob/v19.0.0/CHANGELOG.md) for a list of relevant changes or [compare all changes](https://github.com/cosmos/gaia/compare/v18.1.0...v19.0.0) from last release.
 
 <!-- Add the following line for major releases -->
-Refer to the [upgrading guide](https://github.com/cosmos/gaia/blob/release/<v*.x>/UPGRADING.md) when migrating from `<v-last.x>` to `<v*.x>`.
+Refer to the [upgrading guide](https://github.com/cosmos/gaia/blob/v19.0.0/UPGRADING.md) when migrating from `v18.1.0` to `v19.0.0`.
 
 ## 🚀 Highlights
 
-<!-- Add any highlights of this release -->
+This release brings cosmos-sdk v0.50.x, ibc-go v8.x, interchain-security v5.1.0, and cometbft v0.38.9 to the Cosmos Hub.
+
+Aside from the security and performance upgrades, this release will allow [channel upgrades](https://ibc.cosmos.network/main/ibc/channel-upgrades/) and help relayer incentivization through ibc fees on existing channels.
 
 ## 🔨 Build from source
```

Comment:
Add golang version

Reply:
It doesn't update it, we had to bump to `1.22` for feemarkets.